catroid/src/main/res/values-bs/strings.xml

```diff
@@ -1787,7 +1787,7 @@ needs read and write access to it. You can always change permissions through you
     <string name="formula_editor_logic_comparison">Operatori poređenja</string>
     <string name="formula_editor_object_general">Generalne osobine</string>
     <string name="formula_editor_object_movement">Osobine pokreta</string>
-    <string name="formula_editor_device_visual_detection">Visual sensors</string>
+    <string name="formula_editor_device_face_detection">Visual sensors</string>
```

Comment:
undo this changes, nothing to do here, it will eventually be overwritten by the translations